source/boundary_velocity/interface.cc

```diff
@@ -104,8 +104,8 @@ namespace aspect
       std_cxx11::tuple
       <void *,
       void *,
-      internal::Plugins::PluginList<Interface<2> >,
-      internal::Plugins::PluginList<Interface<3> > > registered_plugins;
+      aspect::internal::Plugins::PluginList<Interface<2> >,
+      aspect::internal::Plugins::PluginList<Interface<3> > > registered_plugins;
```

Comment:
I think that all of the changes to the src files could also be merged independently -- they all seem like good ideas in principle.